rpm/user-managerd.spec

```diff
@@ -0,0 +1,49 @@
+Name: user-managerd
+Version: 0.0.1
+Release: 1
+Summary: Sailfish user manager daemon
+License: BSD
+Source0: %{name}-%{version}.tar.gz
+URL: https://github.com/sailfishos/user-management-daemon.git
+BuildRequires: pkgconfig(Qt5Core)
+BuildRequires: pkgconfig(Qt5DBus)
+BuildRequires: pkgconfig(libuser)
+Requires: systemd
+Requires: sailfish-setup >= 0.1.12
+%description
+%{summary}.
+
+%package devel
+Summary: Sailfish user manager daemon development files
+Requires: user-managerd
+
+%description devel
+%{summary}.
+
+%files
+%defattr(-,root,root,-)
+%{_bindir}/%{name}
+/lib/systemd/system/*.service
+%{_datadir}/dbus-1/system-services/*.service
+%{_sysconfdir}/dbus-1/system.d/*.conf
+
+%files devel
+%{_prefix}/include
```

Comment:
This probably shouldn't contain /usr/include and /usr/lib/pkgconfig directories themselves.